spec/rubocop/cop/style/symbol_array_spec.rb

```diff
@@ -68,6 +68,11 @@
       expect_no_offenses('[:one, :two, :"space here"]')
     end
 
+    # Bug: https://github.com/bbatsov/rubocop/issues/4481
+    it 'does not register an offense in an ambiguous block context' do
+      expect_no_offenses('foo [:bar, :baz] { qux }')
```

Comment:
Ruby can't parse `foo %[] {block}`, but Ruby can parse `foo(%[]) {block}`.(with braces)
Does this cop add an offense for it?